src/Compilers/Core/CodeAnalysisTest/Microsoft.CodeAnalysis.UnitTests.csproj

```diff
@@ -19,6 +19,7 @@
     <ProjectReference Include="..\..\..\Test\Utilities\Portable\Roslyn.Test.Utilities.csproj" />
     <ProjectReference Include="..\..\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.csproj" />
     <ProjectReference Include="..\..\Test\Resources\Core\Microsoft.CodeAnalysis.Compiler.Test.Resources.csproj" />
+    <ProjectReference Include="..\..\Test\Utilities\CSharp\Microsoft.CodeAnalysis.CSharp.Test.Utilities.csproj" />
```

Comment:
>..\..\Test\Utilities\CSharp\Microsoft.CodeAnalysis.CSharp.Test.Utilities.csproj [](start = 31, length = 79)

Do we need this project reference? If it's just to create a `CSharpCompilation`, we should use `new CSharpCompilation(...)` directly instead.

Reply:
Ah, yeah that was why I added it. I originally started by calling `CSharpCompilation` directly, but it was proving tricky to get all the references right. I'll switch back to that way of doing it.